bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -34,7 +34,7 @@ spec:
 
     ### Pre-requisites
     * A Red Hat OpenShift subscription
-    * OCP 4.7 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
+    * OCP 4.8 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
     * [vSphere prequisites](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-prerequisites.md)
```

Comment:
Does it makes sense to include the [cluster and OS pre-requisites ](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md)?

Reply:
It makes sense to me and did cross my mind although we also have that information in our OpenShift docs, will wait for other to weigh in as well.

Reply:
I think we should add them here too

Reply:
We should actually remove this GitHub link if this for the Red Hat operator and add a link to the OpenShift docs. We shouldn't be pointing customers to GitHub docs. However if this is for the community operator, then it is fine to use GitHub links.

Reply:
this is for the Red Hat operator, I will change the link.

Reply:
How would you figure out the link to the new pre-req? https://github.com/openshift/openshift-docs/pull/33929

Reply:
I updated it to the latest vsphere pre-requisite on openShift docs, if we want to point to the entire pre-requisities @EricPonvelle is it safe to assume it will be under https://docs.openshift.com/container-platform/latest/windows_containers/enabling-windows-container-workloads.html ?